packages/api-server/src/methods/error-code.ts

```diff
@@ -13,3 +13,5 @@ export const METHOD_NOT_SUPPORT = -32000;
 export const WEB3_ERROR = -32001;
 
 export const GW_RPC_REQUEST_ERROR = -32002;
+
+export const HEADER_NOT_FOUND_ERROR = -32000;
```

Comment:
```suggestion
export const HEADER_NOT_FOUND_ERROR = -32003;
```
-32000 is already in used.

Reply:
```json
{
    "jsonrpc": "2.0",
    "id": 2,
    "error": {
        "code": -32000,
        "message": "header not found"
    }
}
```

`-32000` is equal to infura response

Reply:
in that sense we should change the other 32000 error to something else.